easylauncher/src/main/kotlin/com/project/starter/easylauncher/filter/DefaultFont.kt

```diff
@@ -0,0 +1,9 @@
+package com.project.starter.easylauncher.filter
+
+import java.awt.Font
+
+val DEFAULT_EASYLAUNCHER_FONT: Font =
+    EasyLauncherFilter::class.java.classLoader
+        .getResourceAsStream("Roboto-Regular.ttf").use {
+            Font.createFont(Font.TRUETYPE_FONT, it)
+        }
```

Comment:
💭 
Knowing that's an IO operation I'd suggest to make it lazy, so it doesn't get loaded when not needed. 
Also I wouldn't like to make it public if it doesn't have to be

```suggestion
internal val DEFAULT_EASYLAUNCHER_FONT: Font = by lazy {
    EasyLauncherFilter::class.java.classLoader
        .getResourceAsStream("Roboto-Regular.ttf").use {
            Font.createFont(Font.TRUETYPE_FONT, it)
        }
}
```